lib/src/io/node.dart

```diff
@@ -8,14 +8,18 @@ import 'dart:js_util';
 
 import 'package:js/js.dart';
 import 'package:node_interop/fs.dart';
-import 'package:node_interop/node_interop.dart';
+import 'package:node_interop/node_interop.dart' hide process;
 import 'package:node_interop/stream.dart';
 import 'package:path/path.dart' as p;
 import 'package:source_span/source_span.dart';
 import 'package:watcher/watcher.dart';
 
 import '../exception.dart';
 import '../node/chokidar.dart';
+import '../logger.dart';
+
+@JS('process')
+external final process;
```

Comment:
I brought the entire `process` instead of just `process.stdout.isTTY`. Not sure if that's the best approach, but all errors related to `process.env` and `isTTY` are now fixed with the changes in this file